test/foundry/OwnableTwoSteps.t.sol

```diff
@@ -48,6 +48,62 @@ contract OwnableTwoStepsTest is TestParameters, TestHelpers, OwnableTwoStepsErro
         ownableTwoSteps.confirmOwnershipTransfer();
         assertEq(ownableTwoSteps.potentialOwner(), address(0));
         assertEq(ownableTwoSteps.owner(), newOwner);
+        assertEq(uint8(ownableTwoSteps.status()), uint8(Status.NoOngoingTransfer));
+    }
+
+    function testRenounceOwnership() public asPrankedUser(_OWNER) {
+        // 1. Initiate renouncement of ownership
+        vm.expectEmit(false, false, false, true);
+        emit InitiateOwnershipRenouncement();
+        ownableTwoSteps.initiateOwnershipRenouncement();
+        assertEq(ownableTwoSteps.potentialOwner(), address(0));
+        assertEq(ownableTwoSteps.earliestOwnershipRenouncementTime(), block.timestamp + ownableTwoSteps.delay());
```

Comment:
I think we should use `_delay` instead of `ownableTwoSteps.delay()` because there can be other issues in the contract that makes delay not 6 hours and it's more affirmative if the expected value is hardcoded.

Reply:
You are right! Let me adjust